lib/components/molecules/RenderTable/RenderTable.story.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import defaultConfig from './RenderTable.mock';
+import RenderTable, { RenderTableVanilla } from '.';
+import Column from '../../atoms/Column';
+
+storiesOf('Molecules', module)
+  .addParameters({ jest: ['RenderTable', 'RenderTableVanilla'] })
+
+  .add('Table Knobs', () => (
+    <RenderTableVanilla {...defaultConfig}>
+      <Column label="Dessert (100g serving)" name="name" />
+      <Column label="Calories" name="calories" />
+      <Column label="Fat (g)" name="fat" />
+      <Column label="Carbs (g)" name="carbs" />
+      <Column label="Protein (g)" name="protein" />
+    </RenderTableVanilla>
+  ))
+  .add('Table', () => (
+    <RenderTable {...defaultConfig}>
+      <Column label="Dessert (100g serving)" name="name" />
```

Comment:
Good work, I like it. One improvement. Column should be a static props of Table so that. 

<Table.Column name="name">Dessert (100g serving)</Table.Column>